lib/lhm/cleanup/current.rb

```diff
@@ -51,7 +51,8 @@ def lhmn_tables_for_origin
       end
 
       def failed_name
-        "lhma_#{Timestamp.new(Time.now)}_#{origin_table_name}_failed"
+        # NOTE: MySQL table names are limited to 64 characters
+        "lhma_#{Timestamp.new(Time.now)}_#{origin_table_name}_failed"[0..63]
```

Comment:
think we should _also_ strip the ms, and probably even the underscores from the timestamp?